unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWNftsShould.cs

```diff
@@ -41,24 +41,24 @@ protected override IEnumerator SetUp()
     [Test]
     public void NftsUsage()
```

Comment:
Please review the test naming here. I don't understand what these are supposed to cover.